src/components/utils/src/qdb_wrapper/sql_database.cc

```diff
@@ -108,5 +108,13 @@ bool SQLDatabase::Backup() {
   return true;
 }
 
+void SQLDatabase::set_path(const std::string& path) {
```

Comment:
@BSolonenko why do you need this getter and setter? As I can see they are not in use

Reply:
@AKalinich-Luxoft @LuxoftAKutsan @AByzhynar  
We have 2 implementation `sql_database.cc`. 
`src/components/utils/src/sqlite_wrapper/sql_database.cc` and 
`src/components/utils/src/qdb_wrapper/sql_database.cc`
When we try build sdl for `linux` we use first implementation. This implementation has getter and setter for `path`. But wen we try build sdk for `QNX` we use second implementation which `haw not` getter and setter for path.